core/metadata.js

```diff
@@ -456,3 +459,33 @@ function buildFile (filePath /*: string */, stats /*: fs.Stats */, md5sum /*: st
   if ((stats.mode & EXECUTABLE_MASK) !== 0) { doc.executable = true }
   return doc
 }
+
+const CONFLICT_REGEXP = new RegExp('-conflict-\\d{4}(?:-\\d{2}){2}T(?:\\d{2}_?){3}.\\d{3}Z')
+
+function withConflictRegExp (prefix /*: string */) /*: RegExp */ {
```

Comment:
Looking at the function signature, one may expect the `prefix` arg (with `string` type) to be the *main subject*. But then the type of a prefix *with* a regexp doesn't seem obvious...

What about `prefixedConflictRegExp(prefix)`?

Reply:
I didn't understand a thing.

Do you want me to explicit the intention with the name of the function? Like:
```js
const myRegExp = createRegExpWithConflictSuffix('docname');
// /docname-conflict-\d{4}-.../
```

Reply:
I'm just a bit confused by `with*` in the current name.
Maybe it's not necessary? E.g.:

```js
function conflictRegExp (basenamePattern /*: string */ ='(.*)') { ... }
````

Reply:
Naming is hard, if you prefer some name rather than others, let's rename it.
`prefixedConflictRegExp` or `withConflictRegExp` or `createRegExpWithConflictSuffix` or `conflictRegExp` are very in common. This function returns a regular expression that matches anything that has the `CONFLICT_PATTERN` at the end of it: `getRegExpWithConflictPattern` or `withConflictPattern`, …